packages/shared/src/components/cards/PostList.tsx

```diff
@@ -78,7 +78,7 @@ export const PostList = forwardRef(function PostList(
           <>
             <Pill
               label="Collection"
-              className="mb-2.5 text-theme-color-cabbage"
+              className="mb-2.5 bg-theme-overlay-float-cabbage text-theme-color-cabbage"
```

Comment:
Wondering if we should add the color option as a prop as you reuse that in 2 places already?

Reply:
I think its fine for now, don't want to complicate it. Refactored pill usages a little so its in less places as well.